frontend/pages/entities/[slug].page.tsx

```diff
@@ -38,6 +49,25 @@ const EntityDetailPage: FC<EntityProps> = ({ entity }: EntityProps) => {
       />
       <ModuleContainer>
         <ModuleDetail module={entity} />
+        {entity.relatedEntities?.map((relatedEntity) => (
```

Comment:
There probably should be a "Related entities" header/label above wherever the related entities are displayed (but only if `!!entity.relatedEntities?.length`).